amf-api-contract/shared/src/main/scala/amf/apicontract/client/scala/model/domain/bindings/googlepubsub/GooglePubSubChannelBinding.scala

```diff
@@ -0,0 +1,94 @@
+package amf.apicontract.client.scala.model.domain.bindings.googlepubsub
+
+import amf.apicontract.client.scala.model.domain.bindings.{BindingVersion, ChannelBinding}
+import amf.apicontract.internal.metamodel.domain.bindings.GooglePubSubChannelBindingModel._
+import amf.apicontract.internal.metamodel.domain.bindings.GooglePubSubSchemaSettingsModel._
+import amf.apicontract.internal.metamodel.domain.bindings.{GooglePubSubChannelBindingModel, GooglePubSubMessageStoragePolicyModel, GooglePubSubSchemaSettingsModel}
+import amf.apicontract.internal.spec.async.parser.bindings.Bindings.GooglePubSub
+import amf.core.client.scala.model.StrField
+import amf.core.client.scala.model.domain.{DomainElement, Linkable, ObjectNode}
+import amf.core.internal.metamodel.Field
+import amf.core.internal.parser.domain.{Annotations, Fields}
+import amf.shapes.client.scala.model.domain.{Key, NodeShape}
+class GooglePubSubChannelBinding(override val fields: Fields, override val annotations: Annotations)
+    extends ChannelBinding
+    with BindingVersion
+    with Key {
+  override protected def bindingVersionField: Field          = BindingVersion
```

Comment:
Should return StrField, not Field

Reply:
As we saw binding version should be Field